packages/types/src/dtos/report/createReport.dto.ts

```diff
@@ -6,12 +6,19 @@ export class CreateReportRequest extends PickType(Report, [
   "text",
 ] as const) {}
 
+export class CreateReportQuery extends PickType(Report, [
+  "postId",
+  "guideId",
+] as const) {}
+
```

Comment:
Is it necessary for postId and guideId to be in QueryParams?
Can't it just be in the request body like other parameters?

Reply:
I include them in one request body now. Also, I separate the function as you guys have suggested. Does it look fine?

Reply:
LGTM